examples/105-crd-simple/README.md

```diff
@@ -0,0 +1,50 @@
+## crd-simple example
```

Comment:
CRD is semantically incorrect in this instance. We are not watching CRDs, but Custom Resources (or CRs) whose creation is possible by previously defining a CRD object.

Reply:
Change description to "Example of monitoring custom resources.".